scss/_patterns_table-of-contents.scss

```
@@ -37,7 +37,7 @@
       }
 
       &:visited {
-        color: $color-link;
+        color: $colors--theme--link-visited;
```

This is overriden on purpose. If we wanted to keep visited links in purple we wouldn't need any special styling here.
But for table of contents we don't want to change link color when visited (because they are just anchors on the page).

```suggestion
        color: $colors--theme--link-default;
```